src/api/contracts.ts

```diff
@@ -173,10 +173,12 @@ async function init(provider: Provider) {
 
     const { address: owlProxyAddress } = deployedContracts.TokenOWLProxy
     deployedContracts.TokenOWL = await contractsMap.TokenOWL.at<Promise<OWLInterface>>(owlProxyAddress)
+
+    // remove Proxy contracts from obj
     delete deployedContracts.DutchExchangeProxy
     delete deployedContracts.TokenOWLProxy
 
-    const oracleAddress = await deployedContracts.DutchExchange.ethUSDOracle()
+    const oracleAddress = await deployedContracts.DutchExchange.ethUSDOracle.call()
```

Comment:
Change to explicitly call `ethUSDOracle` - @Velenir @anxolin could this have any say in the sometimes returned `0x` address? Truffle explicitly wraps non state changing fns with `.call` but maybe something there broke
/shrug

Reply:
will keep this way anyways but idk /shrug